nav2_dwb_controller/dwb_plugins/src/kinematic_parameters.cpp

```diff
@@ -50,6 +50,8 @@ using std::placeholders::_1;
 namespace dwb_plugins
 {
 
+const double EPSILON = 1E-5;
```

Comment:
Please make this a C++ preprocessor directive if its going to be in the global scope